packages/react-environment/src/webb-context/bridge/bridge.ts

```diff
@@ -1,75 +1,51 @@
 
 export class Bridge {
-  private constructor(private readonly configEntry: BridgeConfigEntry) {}
+  private constructor(private bridgeConfig: BridgeConfig) {}
 
-  static from(config: BridgeConfig, bridgeCurrency: BridgeCurrency): Bridge {
-    console.log('bridgeCurrency in Bridge static constructor: ', bridgeCurrency.name);
-    const bridgeConfigEntry = config[bridgeCurrency.name];
-    return new Bridge(bridgeConfigEntry);
+  static from(bridgeCurrency: WebbCurrencyId): Bridge {
+    console.log('WebbCurrencyId in Bridge static constructor: ', bridgeCurrency);
+    const bridgeConfig = bridgeConfigByAsset[bridgeCurrency];
+    return new Bridge(bridgeConfig);
   }
 
   /*
    *  Get the bridge privy pools
    * */
-  get anchors(): BridgeAnchor[] {
-    return this.configEntry.anchors;
+  get anchors() {
+    return this.bridgeConfig.anchors;
   }
 
   /*
    *  Get the bridge currency
    * */
   get currency() {
-    return this.configEntry.asset;
+    return Currency.fromCurrencyId(this.bridgeConfig.asset);
   }
 
   getTokenAddress(chainId: ChainId) {
-    return this.configEntry.tokenAddresses[chainId];
+    return currenciesConfig[this.bridgeConfig.asset].addresses.get(chainId);
   }
 
   /*
-   *  Get all tokens
+   *  Get all Bridge tokens
    * */
-  static getTokens(config: BridgeConfig): BridgeCurrency[] {
-    return Object.values(config).map((i) => i.asset);
-  }
-
-  static getTokensByAddress(config: BridgeConfig, addresses: string[]): BridgeCurrency[] {
-    return Object.values(config)
-      .filter((cfgEntry) => {
-        const tokenAddress = Object.keys(cfgEntry.tokenAddresses).filter((key) => {
-          const bridgeEntry = cfgEntry.tokenAddresses[key as unknown as ChainId]!;
-          return addresses.includes(bridgeEntry);
-        });
-        return tokenAddress.length > 0;
-      })
-      .map((cfg) => cfg.asset);
+  static getTokens(): Currency[] {
+    const bridgeCurrenciesConfig = Object.values(currenciesConfig).filter((i) => i.type == CurrencyType.BridgeCurrency);
+    return bridgeCurrenciesConfig.map((config) => {
+      return Currency.fromCurrencyId(config.id);
+    });
   }
 
   /*
-   *  Get tokens for a given chain
+   *  Get all Bridge tokens for a given chain
    * */
-  static getTokensOfChain(config: BridgeConfig, chainId: ChainId): BridgeCurrency[] {
-    const tokens = Bridge.getTokens(config);
+  static getTokensOfChain(chainId: ChainId): Currency[] {
+    const tokens = Bridge.getTokens();
     return tokens.filter((token) => token.hasChain(chainId));
   }
-
-  /*
-   *  Get tokens that all that supports given chains
-   * */
-  static getTokensOfChains(config: BridgeConfig, chainIds: ChainId[]): BridgeCurrency[] {
-    const tokens = Bridge.getTokens(config);
-    return tokens.filter((token) => {
-      for (let chainId of chainIds) {
-        if (!token.hasChain(chainId)) {
-          return false;
-        }
-      }
-      return true;
-    });
-  }
```

Comment:
@nepoche  I see this can be a useful utility to get tokens that are among more than one chain, but I support removing it if we don't rely on static configuation to tell if the asset is Bridgeable

Reply:
This method wasn't being used anywhere so I removed it. We can always add back a similar method if it is useful in the future.